front/logout.php

```diff
@@ -45,6 +45,9 @@
     $CFG_GLPI["ssovariables_id"] > 0
     && strlen($CFG_GLPI['ssologout_url']) > 0
 ) {
+    Session::destroy();
```

Comment:
As the same code is present at the end of the file, maybe we can just have these at the top (before the first if).
You just need to check if any path requires to not destroy session (i guess no)

Reply:
from what I see no

Reply:
same for cookie

Reply:
Ok, i move cookie to top too.